src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>To-do List</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Domine&family=Inter:wght@400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <script defer="defer" src="main.js"></script>
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div class="container">
+        <section class="list-title flex-spaced">
+          <p class="title">Today's To Do</p>
+          <i id="reset" class="ptr"></i>
+        </section>
+        <section class="addlist">
+          <input type="text" placeholder="Add to your list..." />
+          <i id="enter" class="ptr"></i>
+        </section>
+        <div class="list-container">
+          <section class="lists">
+            <div class="list">
+              <input type="checkbox" id="single-list" />
+              <label for="single-list">wash the dishes </label>
+            </div>
+            <i id="selection" class="ptr"></i>
+          </section>
+          <section class="lists">
+            <div class="list">
+              <input type="checkbox" id="single-list" />
+              <label for="single-list">complete To Do list project </label>
```

Comment:
- [ ] On page load, the task array should rendered dynamically to the DOM. The tasks should appear in order of the `index` number for each task. Kindly implement this as well